src/DotNet.Testcontainers/Containers/Configurations/Databases/CouchbaseTestcontainerConfiguration.cs

```diff
@@ -0,0 +1,101 @@
+namespace DotNet.Testcontainers.Containers.Configurations.Databases
+{
+  using System;
+  using System.IO;
+  using Abstractions;
+  using OutputConsumers;
+  using WaitStrategies;
+
+  public sealed class CouchbaseTestcontainerConfiguration : TestcontainerDatabaseConfiguration
+  {
+    internal const string CouchbaseImage = "mustafaonuraydin/couchbase-testcontainer:6.0.0";
```

Comment:
private

Reply:
Unfortunately the official image is not providing initialize cluster definition, create user, create bucket etc. On first use you should configure Couchbase over web ui or couchbase-cli. My image is using official image and running installation script over official image with necessary installment script.
My offer is, let's open docker id  like "dotnet-testcontainers" or "testcontainers". After that we can push this image to that repository. And also, we can create small git project containing files which is i wrote Dockerfile and entrypoint.sh . After that this image will be open to contribute and source code will be depended to dotnet-testcontainers project. Or adding docker image build files into dotnet-testcontainers project is also good.

Reply:
Maybe we can add a comment to the image what it contains in addition. If you like you can also set-up your own GitHub Repository for it.